### PR TITLE
Fixes/#113 no ocgt generators

### LINKED ISSUE #113: No OCGT generators

### PR DESCRIPTION
**Pull request by CarlosEpia**
Fixes #113. Already tested for SH.

### CODE REVIEW

File: src/egon/data/datasets/electrical_neighbours.py [hunk 1]
```diff
@@ -1536,6 +1520,14 @@ def insert_generators_sq(gen_sq=None, scn_name="status2019"):
     # Add marginal costs
     list_gen_sq = add_marginal_costs(list_gen_sq)
 
+    # assign marginal cost for OCGTs
+    scn_params_gas = get_sector_parameters("gas", scn_name)
+    efficiency_ocgt = scn_params_gas["efficiency"]["OCGT"]
+    list_gen_sq.loc[list_gen_sq.carrier=="OCGT","marginal_cost"] = (
+        scn_params_gas["marginal_cost"]["OCGT"]
+        / efficiency_ocgt
+    )
```
Comment (ClaraBuettner) [inline src/egon/data/datasets/electrical_neighbours.py]: These costs do not include fuel costs for the gas

File: src/egon/data/datasets/electrical_neighbours.py [hunk 2]
```diff
@@ -1494,28 +1494,12 @@ def insert_generators_sq(gen_sq=None, scn_name="status2019"):
...
-                "lignite",
-                "coal",
-                "oil",
-                "nuclear",
-            ]
-        ),
-    ]
+
+    # Filter generators modeled as storage and geothermal
+    gen_sq = gen_sq.loc[:,~gen_sq.columns.isin(["Hydro Pumped Storage", "reservoir", "geo_thermal"])]
```
Comment (ClaraBuettner) [inline src/egon/data/datasets/electrical_neighbours.py]: We need the reservoir power plants here

File: src/egon/data/datasets/electrical_neighbours.py (hunk 1 above)
Comment (CarlosEpia) [reply src/egon/data/datasets/electrical_neighbours.py]: Corrected in 59db364

File: src/egon/data/datasets/electrical_neighbours.py (hunk 2 above)
Comment (CarlosEpia) [reply src/egon/data/datasets/electrical_neighbours.py]: Corrected in 59db364

File: src/egon/data/datasets/electrical_neighbours.py [hunk 3]
```diff
@@ -1399,21 +1399,21 @@ def map_carriers_entsoe():
...
         "Geothermal": "geo_thermal",
         "Hydro Pumped Storage": "Hydro Pumped Storage",
         "Hydro Run-of-river and poundage": "run_of_river",
         "Hydro Water Reservoir": "reservoir",
-        "Marine": "marine",
+        "Marine": "Others",
         "Nuclear": "nuclear",
         "Other": "others",
-        "Other renewable": "others",
+        "Other renewable": "Others",
```
Comment (ClaraBuettner) [inline src/egon/data/datasets/electrical_neighbours.py]: Others should be written like this: 'others'

File: src/egon/data/datasets/electrical_neighbours.py (hunk 1 above)
Comment (ClaraBuettner) [reply src/egon/data/datasets/electrical_neighbours.py]: But the efficiency is missing isn't it? As we discussed yesterday the CH4 costs are given in EUR/MWh_CH4

**Comment by CarlosEpia** [comment]
The efficiencies of some power plants were included and now their marginal costs increased. They are now to a certain extent comparable with values available in independent reports: https://static.agora-energiewende.de/fileadmin/Projekte/2019/Jahresauswertung_2019/A-EW_German-Power-Market-2019_Summary_EN.pdf

image

File: src/egon/data/datasets/scenario_parameters/parameters.py [hunk 4]
```diff
@@ -314,27 +314,32 @@ def electricity(scenario):
...
+            ] / read_costs(costs, "OCGT", "efficiency")
             + global_settings(scenario)["co2_costs"]
             * global_settings(scenario)["co2_emissions"][
                 "other_non_renewable"
             ],
             "lignite": global_settings(scenario)["fuel_costs"]["lignite"]
+            / read_costs(costs, "lignite", "efficiency")
             + read_costs(costs, "lignite", "VOM")
             + global_settings(scenario)["co2_costs"]
             * global_settings(scenario)["co2_emissions"]["lignite"],
```
Comment (ClaraBuettner) [inline src/egon/data/datasets/scenario_parameters/parameters.py]: The Co2 costs were not changed, which would mean that these costs have been in EUR / MWh_el. Is that correct?

Comment (CarlosEpia) [reply src/egon/data/datasets/scenario_parameters/parameters.py]: I thought so but they are actually based on the CO2 emission of the source, so the efficiency must be considered. Already corrected in the last commit.

**Review by ClaraBuettner — APPROVED** [review]

**Pull request merged**